docs/release_notes.md

```diff
@@ -33,6 +37,11 @@
 - Fixed import of the constraint weight arrays on sparse arrays. *AdonisFX-1155*
 - Fixed a bug that caused geometry sliding constraints to not find valid triangles to slide on initialisation. *AdonisFX-1165*
 - Fixed paint flood operation not normalising the weights properly. *AdonisFX-1146*
+- Fixed AdonisFX Paint Tool initialization of simulated nodes with intermediate nodes. *AdonisFX-1184*
+
+### Known Limitations
+
+- The activations debugger in AdnSimshape is limited to the Parallel and Serial evaluation modes. *AdonisFX-1026*
```

Comment:
Leaving a comment here just in case we have more limitations to add. I'm thinking

Reply:
Should we add the limitation abou the volume preservation issue?

Reply:
We also encountered that the system might become unstable when we have a very high mass value in DampVelocities

Reply:
I know this might not be the right way of exposing the limitation but should we mention something about the stiffness ranges?

Reply:
Also is it considered a limitation to not be able to have separate stiffness per constraint?

Reply:
> Should we add the limitation about the volume preservation issue?

Added

Reply:
Resolving this comment!